commands/theme/preview.ts

```diff
@@ -207,12 +239,6 @@ exports.builder = yargs => {
     type: 'string',
     requiresArg: true,
   });
-  yargs.option('notify', {
```

Comment:
Technically removing this makes it a breaking change. We're queuing up a major release in the `next` branch. If you merge this into that branch it should make getting this out alongside everything else nice and easy.